### PR TITLE
Fix typo's

### PR DESCRIPTION
**Pull request by OrangeTux**
Signed-off-by: Auke Oosterhoff <auke@orangetux.nl>

### CODE REVIEW

File: 01.Getting-started/04.Deploy-to-physical-devices/docs.md [hunk 1]
```diff
@@ -120,12 +120,12 @@ Gateway=$IP_OF_MENDER_SERVER_FROM_DEVICE
...
-  
-> [Network]  
-> Address=192.168.10.2  
-> Gateway=192.168.10.1  
+> [Match]
+> Name=eth0
+
+> [Network]
+> Address=192.168.10.2
+> Gateway=192.168.10.1
```
Comment (drewmoseley) [inline 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: Are these whitespace changes necessary?  They are obviously harmless but I'm just curious if there is something else wrong with them?

File: 01.Getting-started/04.Deploy-to-physical-devices/docs.md [hunk 2]
```diff
@@ -202,7 +202,7 @@ the steps needed to edit the root file system contained in this Artifact are:
 [start_autoupdate_beaglebone_release_1_x.x.x.mender]: #
 
 ```bash
-mkdir beaglebone_release_1 && tar -C beaglebone_release_1 -xvf beaglebone_release_1_master.mender
+mkdir beaglebone_release_1 && tar -C beaglebone_release_1 xvf beaglebone_release_1_master.mender
```
Comment (drewmoseley) [inline 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: I think tar works with or without the '-' for options.  At least for gnu tar.
Are there versions of tar that will not work with the leading '-' character?

File: 01.Getting-started/04.Deploy-to-physical-devices/docs.md (hunk 1 above)
Comment (OrangeTux) [reply 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: The whitespaces aren't necessary. My editor removed them automatically. I can undo these edits if you want.

File: 01.Getting-started/04.Deploy-to-physical-devices/docs.md (hunk 2 above)
Comment (OrangeTux) [reply 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: Not that I'm aware of. I docs are inconsistent with the `-`. Sometimes it's used, sometimes it isn't. I tried to make it consistent.

Comment (drewmoseley) [reply 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: I see that now.  Thanks.

File: 01.Getting-started/04.Deploy-to-physical-devices/docs.md (hunk 1 above)
Comment (drewmoseley) [reply 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: No need.  I was just curious.

**Comment by drewmoseley** [comment]
Looks good to me.   Thanks.
I'll also cherry-pick to older releases.

**Pull request merged**